code/modules/jobs/job_types/clown.dm

```diff
@@ -57,3 +57,5 @@
 
 	H.fully_replace_character_name(H.real_name, pick(GLOB.clown_names)) //rename the mob AFTER they're equipped so their ID gets updated properly.
 	H.dna.add_mutation(CLOWNMUT)
+	for(var/datum/mutation/human/clumsy/M in H.dna.mutations)
+		M.mutadone_proof = TRUE
```

Comment:
This is more like a band aid. Basically mutadone can remove all mutations, even ones created through stuff like wizardry. This is an oversight.

https://github.com/tgstation/tgstation/blob/e4e3ff3c49abc8836c82bb47e53f71528eea65c5/code/modules/reagents/chemistry/reagents/medicine_reagents.dm#L893
Pass the parameter list(MUT_NORMAL, MUT_EXTRA) in remove_all_mutations

Reply:
M.dna.remove_all_mutations(MUT_NORMAL, MUT_EXTRA, mutadone = TRUE)
like this?

Reply:
M.dna.remove_all_mutations(list(MUT_NORMAL, MUT_EXTRA), TRUE)

Reply:
ok done